docs/feature-gates.md

```diff
@@ -89,8 +87,7 @@ for more information about EndpointSlice. If this feature is enabled but the End
 
 #### Requirements for this Feature
 
-- EndpointSlice v1 API is available (Kubernetes version >=1.21).
-- `AntreaProxy` is enabled.
+EndpointSlice v1 API is available (Kubernetes version >=1.21).
```

Comment:
I suppose if use set the option `antreaProxy.enable` to false, this feature still won't work? If so, we should add another line like "`antreaProxy.enable` is true"?
@tnqn any suggestion here?

Reply:
I think we can keep a mention to AntreaProxy being enabled.